app/views/schools/add_participants/still_cannot_find_their_details.html.erb

```diff
@@ -13,29 +12,37 @@
     <p class="govuk-body">
       This could be because the information does not match their Teaching Regulation Agency (TRA) record
     </p>
-    <h2 class="govuk-heading-m">
-     Contact our support team as soon as possible
-    </h2>
-    <p class="govuk-body">
-      We need to do a manual check of the records.
-    </p>
-    <p class="govuk-body">
-      Email us at: <%= govuk_mail_to "continuing-professional-development@digital.education.gov.uk",
-                                     "continuing-professional-development@digital.education.gov.uk" %>
-    </p>
+
+    <%= govuk_summary_list do |summary_list| %>
+      <% summary_list.row do |row| %>
+        <% row.key { "TRN" } %>
+        <% row.value { @wizard.trn } %>
+        <% row.action(text: "Change",
+                      visually_hidden_text: "TRN",
+                      href: @wizard.change_path_for(step: :trn)) %>
+      <% end %>
+
+      <% summary_list.row do |row| %>
+        <% row.key { "Date of birth" } %>
+        <% row.value { @wizard.date_of_birth.to_date.to_s(:govuk) } %>
+        <% row.action(text: "Change",
+                      visually_hidden_text: "date of birth",
+                      href: @wizard.change_path_for(step: :date_of_birth)) %>
+      <% end %>
+
+      <% summary_list.row do |row| %>
+        <% row.key { "National Insurance number" } %>
+        <% row.value { @wizard.nino } %>
+        <% row.action(text: "Change",
+                      visually_hidden_text: "National Insurance number",
+                      href: @wizard.change_path_for(step: :nino)) %>
+      <% end %>
+    <% end %>
+
+    <h3 class="govuk-heading-m">If you are sure this information is correct</h3>
     <p class="govuk-body">
-      Include their:
-    </p>
-    <ul class="govuk-list govuk-list--bullet">
-      <li>full name</li>
-      <li>teacher-reference-number (TRN)</li>
-      <li>date of birth</li>
-    </ul>
-
-    <p>
-      <%= govuk_link_to "View your ECTs and mentors",
-                        schools_participants_path,
-                        class: "govuk-link govuk-link--no-visited-state" %>
+      Contact us for help to register this <%= @wizard.ect_participant? ? "ECT" : "mentor" %> at your school: <%= render MailToSupportComponent.new %>
     </p>
+    <%= govuk_link_to "Return to your ECTs and mentors", schools_participants_path, class: "govuk-link govuk-link--no-visited-state" %>
```

Comment:
We can [lean on the link helper](https://govuk-components.netlify.app/helpers/link/#other-link-styles) for these classes:

```suggestion
    <%= govuk_link_to "Return to your ECTs and mentors", schools_participants_path, no_visited: true %>
```